packages/boba/register/addresses/layerZeroTestnet.json

```diff
@@ -0,0 +1,77 @@
+{
+  "Layer_Zero_Protocol": {
```

Comment:
I guess these change also - do we have those deployed on Sepolia already? @Boyuan-Chen

Reply:
No